stable/search-prod/values.yaml

```diff
@@ -5,7 +5,7 @@
 # Copyright (c) 2020 Red Hat, Inc.
 # Copyright Contributors to the Open Cluster Management project
 
-org: open-cluster-management
+org: stolostron
```

Comment:
I'm not sure if we are changing this one right now. It affects the name of the resources deployed.
The [console-chart PR](https://github.com/stolostron/console-chart/pull/176) and [application-chart PR](https://github.com/stolostron/application-chart/pull/66) didn't update this.